tests/test_invoices.py

```diff
@@ -181,7 +181,7 @@ def test_invoice_routeboost_private(node_factory, bitcoind):
     # Attach public channel to l1 so it doesn't look like a dead-end.
     l0 = node_factory.get_node()
     l0.rpc.connect(l1.info['id'], 'localhost', l1.port)
-    scid = l0.fund_channel(l1, 2 * (10**4))
+    scid = l0.fund_channel(l1, 2 * (10**5))
```

Comment:
For future reference: commit message should say *why* this is done; in this case "we're about to increase the minimum channel capacity".

Reply:
Thanks, I added this to the commits log message, as the standard commit message is a bit too short reasoning.